batch/2017-01-01.4.0/examples/NodeGet_SelectClause.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "poolId": "poolId",
+    "nodeId": "tvm-1695681911_1-20161122t193202z",
+    "api-version": "2017-01-01.4.0",
+    "client-request-id": "00000000-0000-0000-0000-000000000000",
+    "ocp-data": "Fri, 17 Feb 2017 00:00:00 GMT",
+    "$select": "id%2Cstate"
```

Comment:
Why do we show a $select example specifically for this one?  Or does this sample relate to a 'how to do a select' page rather than the Get Node reference page?